index.js

```diff
@@ -31,22 +35,47 @@
 // // => 11
 // console.log("%c----------", "color: red") 
 
+// function drinkWater(current_thirst_level) {
+//   if (current_thirst_level -= 1){
+//     return "Man I sure am thirsty"
+//   }
+//   else{
+//     return "Ahh that hits the spot"
+//   }
+// }
+
+function drinkWater(currentThirstLevel) {
+  console.log("Man, I sure am thirsty")
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel - 1
+}
+console.log(drinkWater(12))
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 4", "color: red") 
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum){
+  return "same same"
+  }
+  else if (num == maybeNum){
+    return "same same (but different)"
+  }
+  else return "different"
+  }
+  
```

Comment:
Nice! Another possible solution would be to convert the string to a number with parseInt to avoid == :

`} else if (num === parseInt(maybeNum)) {`